### ISSUE TITLE
Wrong agent is used on run page

### ISSUE THREAD

**Issue by Sliman4** [nearai/nearai#577]
https://app.near.ai/agents/slimedragon.near/memecoin-trading-agent/0.4/run if you send "swap 10 near for intear", it will output something like "Token you provided doesn't belong to the list of available tokens", even though there is no such thing as a list of available tokens in this agent's code. I believe it's from zavodil.near/swap-agent, which it's a fork of. On the page, the name of the agent is incorrect ("swap agent" instead of "Memecoin trading agent"), but description is correct.
[Screencast_20241115_163922.webm](https://github.com/user-attachments/assets/2f1ad3c3-d05e-4134-b22b-2fdaae5a09c6)

**Comment by gabehamilton** [comment]
Looks like version 0.4 has an issue in utils.py line 103 with the "s used in 
`url = f"https://smartrouter.ref.finance/findPath?amountIn={amount_in}&tokenIn={state.token_in_data["account_id"]}&tokenOut={state.token_out_data["account_id"]}&pathDeep=3&slippage=0.001"
`

The 'Swap agent' text seen when the agent loads appears to match what is in the metadata.json welcome.title.

**Comment by Sliman4** [comment]
My bad haha, and I'm bad at python and maybe didn't notice it. Though I think it should error out if the syntax is incorrect, no idea why it tried to execute a different agent. Maybe I didn't notice that the list is still there or something, it was almost impossible to debug, so after a few hours I gave up and just assumed it's a bug in frontend or something.

Now when I'm trying this agent with the same version and same first message, it doesn't send any messages at all, probably because of the " error, but for some reason it did a different thing when I recorded the video.

I won't keep developing this agent and won't be able to confirm a fix, so feel free to close this if you think it works correctly now
